### ISSUE TITLE
Not necessary to add GitHub token to the .travis.yml file

### ISSUE THREAD

**Issue by ashalkhakov** [steveklabnik/automatically_update_github_pages_with_travis_example#14]
It seems that the GitHub token can be added to the repository settings in Travis dashboard: pick a repo, click on 'More options', then 'Settings', 'Environment variables', then add GH_TOKEN environment variable. There is no need to encrypt it, just don't mark the 'Display value in build log'.

Thank you for the write-up!

**Comment by steveklabnik** [comment]
Yes, adding some text about this would be great! I'd happily take a PR for this.
